cmd/fieldnamedocscheck/field_name_docs_check.go

```diff
@@ -71,7 +71,8 @@ func main() {
 
 func checkFieldNameAndDoc(structName, fieldName, doc string, typesMap kubeTypesMap) bool {
 	rc := false
-	visited := sets.Set[string]{}
+	// sets.Set[string]() immediately opens a map and resets map and sometimes empty map go to panic, change to New
+	visited := sets.New[string]() 
```

Comment:
`sets.New[string]()` results in the same thing as `sets.Set[string]{}`

I don't think this is a useful change, or a correct comment

/close